sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1145,15 +1147,16 @@ object functions {
   def randn(seed: Long): Column = withExpr { Randn(seed) }
 
   /**
-   * Generate a column with i.i.d. samples from the standard normal distribution.
+   * Generate a column with independent and identically distributed (i.i.d.) samples from
+   * the standard normal distribution.
    *
    * @group normal_funcs
    * @since 1.4.0
    */
   def randn(): Column = randn(Utils.random.nextLong)
 
   /**
-   * Partition ID of the Spark task.
```

Comment:
This one was missed in https://github.com/apache/spark/commit/02f203107b8eda1f1576e36c4f12b0e3bc5e910e and https://github.com/apache/spark/commit/a08463b1d32348a81d0f148dfaf22741d5c23b1a